packages/@aws-cdk/region-info/lib/fact.ts

```diff
@@ -23,6 +23,23 @@ export class Fact {
     return regionFacts && regionFacts[name];
   }
 
+  /**
+   * Retrieve a fact from the Fact database. (retrieval will fail if the specified region or
+   * fact name does not exist.)
+   *
+   * @param region the name of the region (e.g: `us-east-1`)
+   * @param name the name of the fact being looked up (see the `FactName` class for details)
+   */
+  public static getFact(region: string, name: string): string {
```

Comment:
Names starting with `get` are not ideal (they won't translate very cleanly in other supported languages like Java - where it'll collide with a possible future static property getter).

How about naming it `require`? (This is just a suggestion - if you have a better idea that doesn't start with `get`, that should be fine).

Reply:
I like your suggestion, and that escaped my mind. Updated to `requireFact`.